docs/tutorials/writing-sd-card-image-etcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing an SD Card Image Using Etcher (Cross-Platform)
+subject: SD Card Image
+---
+
+1. Download and install Etcher from [their website](http://etcher.io).
+2. Launch Etcher after it has finished installing.
+
+   <div>{% include screenshot.html source="/images/etcher/home.png" caption="Etcher home screen" %}</div>
```

Comment:
The problem is that we don't want it to add paragraph tags. When it does, because you can't have block-level elements in paragraphs, the inner paragraphs and other content get messed up.